Code/Editor/EditorFramework/EditorApp/EditorApp.moc.h

```diff
@@ -265,21 +265,21 @@ private Q_SLOTS:
   ezBitflags<StartupFlags> m_StartupFlags;
   ezDynamicArray<ezString> m_DocumentsToOpen;
 
-  ezSet<ezString> s_RestartRequiredReasons;
-  ezSet<ezString> s_ReloadProjectRequiredReasons;
+  ezSet<ezString> m_RestartRequiredReasons;
+  ezSet<ezString> m_ReloadProjectRequiredReasons;
 
   ezPluginBundleSet m_PluginBundles;
 
   void SaveRecentFiles();
   void LoadRecentFiles();
 
-  ezRecentFilesList s_RecentProjects;
-  ezRecentFilesList s_RecentDocuments;
+  ezRecentFilesList m_RecentProjects;
+  ezRecentFilesList m_RecentDocuments;
 
-  int m_iArgc = 0;
+  int m_Argc = 0;
   QApplication* m_pQtApplication = nullptr;
   ezLongOpControllerManager m_LongOpControllerManager;
-  ezEditorEngineProcessConnection* s_pEngineViewProcess;
+  ezEditorEngineProcessConnection* m_pEngineViewProcess;

```

Comment:
Can we enforce that pointers are nullptr initialized ? It's the one thing that IMO should never be left uninitialized.

Reply:
Yes, we csn do that with clang-tidy.